content/kubernetes/reference/supported_k8s_distributions.md

```diff
@@ -36,16 +36,16 @@ Each release of the Redis Enterprise operator is thoroughly tested against a set
 - Any distribution not listed below is not supported for production workloads.
 
 
-| **Kubernetes version**  | 1.19       | 1.20       | 1.21       | 1.22       | 1.23       |
-|:------------------------|:----------:|:----------:|:----------:|:----------:|:----------:|
-| Community Kubernetes    | deprecated | deprecated | supported  | supported  | supported* |
-| Amazon EKS              | supported  | supported  | supported  |            |            |
-| Azure AKS               |            |            | supported  | supported  | supported* |
-| Google GKE              | supported  | supported  | supported  | supported  |            |
-| Rancher 2.6             | supported  | supported  | supported  | supported  |            |
+| **Kubernetes version**  | 1.19       | 1.20       | 1.21       | 1.22       | 1.23       | 1.24       |
+|:------------------------|:----------:|:----------:|:----------:|:----------:|:----------:|:----------:|
+| Community Kubernetes    |            |            | deprecated | supported  | supported  | supported* |
```

Comment:
Vanilla Kubernetes 1.20 should be reinstated here (as deprecated), since in the previous release it was inconsistently mentioned as supported (in k8s-docs) and deprecated (official docs). Let's take the conservative approach and have it deprecated for yet another round before being removed. In terms of actual support - it was tested and certified throughout this release similarly to the rest of the supported versions, so we're good with that.